src/routes/__tests__/events.test.js

```diff
@@ -645,3 +646,83 @@ describe('POST /api/event/:id/volunteers', () => {
             });
     });
 });
+
+describe('Fetching events by id', () => {
```

Comment:
I did not run these tests since we are not doing testing anymore. I hope they are working correctly though.

Reply:
We ran them and they passed